modules/aws_ecs/temporal/variables.tf

```diff
@@ -206,3 +206,7 @@ variable "task_propagate_tags" {
   default     = "TASK_DEFINITION"
 }
 
+variable "service_discovery_namespace" {
+  type        = string
+  description = "Service discovery namespace DNS name for Retool ECS cluster."
```

Comment:
Should this have a default?

Reply:
This needs to be passed in from the calling module (aws_ecs), where it's computed as a local value.